test/integration/api.bats

```diff
@@ -43,20 +43,21 @@ function teardown() {
 }
 
 @test "docker attach through websocket" {
-	#FIXME: Broken
-	skip
-
 	CLIENT_API_VERSION="v1.17"
 	start_docker 2
 	swarm_manage
 
 	#create a container
 	run docker_swarm run -d --name test_container busybox sleep 1000
+	[ "$status" -eq 0 ]
 
 	# test attach-ws api
 	# jimmyxian/centos7-wssh is an image with websocket CLI(WSSH) wirtten in Nodejs
 	# if connected successfull, it returns two lines, "Session Open" and "Session Closed"
 	# Note: with stdout=1&stdin=1&stream=1: it can be used as SSH
+	docker_host pull jimmyxian/centos7-wssh
```

Comment:
Oh yeah!

Well, instead of pulling, can you just check `$output` instead of `$lines`?